text/0034-telemetry-policies.md

```diff
@@ -0,0 +1,146 @@
+- Title: Telemetry Policies
+- Start Date: 2021-04-16
+- Specification PR: [#34](https://github.com/meilisearch/specifications/pull/34)
+- MeiliSearch Tracking-Issues: [transplant/#181](https://github.com/meilisearch/transplant/issues/181)
+
+# Telemetry Policies
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Telemetry concerns the possibility of depositing markers within a system in order to make segments of it. Quantitative data as data metrics are used to discover and verify user behavior.
+
+We would like to have more telemetry to better understand our users in order to improve their usage and feel with MeiliSearch.
+
+We also use Sentry to trace the errors that occur when using MeiliSearch.
+
+### II. Motivation
+
+We need to make informed decisions based on data instead of instincts and gut feel. Lack of data does not say anything about how correct or wrong are our intuitions. Lack of data means we did not make testable hypotheses. We will now support our decisions with data.
+
+> 🔒 Note that we strictly don't want to collect private information by default (ip address, email, website url, etc..).
+
+We want to simplify the disabling of telemetry and better inform users about what we want to track.
+
+### III. Additional Materials
+N/A
+
+### IV.Explanation
+
+Following this issue and with the beginning of a product Team we asked ourselves what we wanted to do about telemetry. https://github.com/meilisearch/transplant/issues/141.
+
+We will keep telemetry enabled by default while leaving the option for users to disable it. See Impact on Documentation part.
+
+> Some new features will have telemetry data points to check the assumptions and measure objectives achievements.
+
+Each new feature specification should list data points to be collected and measured. Obviously the specification must explain why these data points are necessary for relation to the feature.
+
+We will simplify the deactivation of all our telemetries in one action. To do that, we will remove the Sentry specific flag `MEILI_NO_SENTRY` to merge its use into the `MEILI_NO_ANALYTICS` flag. This means that its activation and deactivation will be shared with analytics within the `MEILI_NO_ANALYTICS` flag. By default, Sentry metrics will stay activated.
```

Comment:
From the perspective of the docs, it's always a bit awkward to tell a user that `true` deactivates something. Wouldn't it be more intuitive to have a flag `MEILI_ANALYTICS` whose default is `true` and which can be deactivated by setting it to `false`?

Reply:
Nice catch @guimachiavelli! @tamo also brought up the same feedback yesterday and it's something we'll deal with in a future specification probably the one that will tackle the convention of our configuration variables and option flags as a whole!